# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296987205 18000
#      Sun Feb 06 05:13:25 2011 -0500
# Node ID 7b153b1af227df4f890fc8390b2f8dae4eceb7ee
# Parent  d1e48a4a9a0b66c730f936ff32e8e74232e8dc6c
PROJECTS, README.devel: use maintainers@octave.org instead of octave-maintainers@octave.org

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2011-02-06  John W. Eaton  <jwe@octave.org>
+
+	* PROJECTS, README.devel: Use maintainers@octave.org instead of
+	octave-maintainers@octave.org.
+
 2010-02-05  Rik  <octave@nomad.inbox5.com>
 
 	* PROJECTS: Point to maintained list on Octave Wiki.
 
 2010-02-05  Rik  <octave@nomad.inbox5.com>
 
 	* mkoctfile.in, octave-config.in: Update usage strings.
 
diff --git a/PROJECTS b/PROJECTS
--- a/PROJECTS
+++ b/PROJECTS
@@ -1,11 +1,11 @@
 Octave PROJECTS
 ===============
 
 A list of proposed projects is maintained at: 
              
              http://wiki.octave.org/wiki.pl?Projects
 
 If you start working steadily on a project, please let
-octave-maintainers@octave.org know.  We might have information that
-could help you.  You should also read the Contributing Guidelines
-chapter in the Octave manual.
+maintainers@octave.org know.  We might have information that could help
+you.  You should also read the Contributing Guidelines chapter in the
+Octave manual.
diff --git a/README.devel b/README.devel
--- a/README.devel
+++ b/README.devel
@@ -14,17 +14,17 @@ more than one release.
 
 Please DO report other bugs in the development releases as soon as you
 find them.  Bugs should be reported to the bug tracker at
 'http://bugs.octave.org'.  Please read read the bug reporting
 guidelines (http://www.gnu.org/software/octave/bugs.html) before
 submitting an item.
 
 If you have a fix for a bug, or an enhancement to submit, send your
-patch to octave-maintainers@octave.org or submit it to the patch
+patch to maintainers@octave.org or submit it to the patch
 tracker at 'http://savannah.gnu.org/patch/?group=octave'. 
 
 By adhering to the following guidelines you can minimize the work that
 Octave maintainers need to do to apply your patch.  Maintaining Octave
 is a lot of work in the best of circumstances, and we can't keep up
 unless you do your best to help.
 
    * Send an explanation with your changes of what problem they fix or
